### ISSUE TITLE
file load version conflict

### ISSUE THREAD

**Issue by sagarhz** [torinmb/mediapipe-touchdesigner#49]
Hey there! I just wanted to express my gratitude for the amazing work you all have done. I really appreciate the fact that you share it with others. However, I encountered an error while trying to open the project file. Although I accepted the prompt to open the project, the MediaPipe seems to be stuck in the loading state and is not working properly.

![image](https://github.com/torinmb/mediapipe-touchdesigner/assets/135154493/5f2eac7c-ba35-4c85-a420-af25ee1b68a7)
![image](https://github.com/torinmb/mediapipe-touchdesigner/assets/135154493/66d8e037-9fe0-4c2f-8192-c919c04cfa1d)

**Comment by domisjustanumber** [comment]
Hi @sagarhz thanks for the kind words 😊

As the message says, we have only tested the componemt in 2022.33910 (and later) versions of TouchDesigner. If you update your installation to a newer version, it should work.

**Comment by sagarhz** [comment]
Thanks for your response. As you said, I've tried with the newer version, and although the error at the beginning is gone, the problem has stayed the same.🥲🫠

![image](https://github.com/torinmb/mediapipe-touchdesigner/assets/135154493/0ebb7426-1747-4abc-870e-6af54277e5fb)

**Comment by domisjustanumber** [comment]
Sorry it's still not loading. This usually happens when the MediaPipe component can't load the webcam source you have selected.

Can you confirm which webcam you have selected?

If you are using SpoutCam, the video source being fed to SpoutCam needs to be 720p otherwise that can also cause it to fail

**Comment by sagarhz** [comment]
It's my default laptop webcam, and as I can't use any other webcam right now, is there any solution?
![image](https://github.com/torinmb/mediapipe-touchdesigner/assets/135154493/84c0c03f-ce1b-4bf8-998f-18ba3578d1e3)

**Comment by domisjustanumber** [comment]
We just released a new version that has more debug and fixes a few bugs around webcams not loading, would you be able to try it and see what happens? https://github.com/torinmb/mediapipe-touchdesigner/releases/tag/v0.3.2

**Comment by sagarhz** [comment]
I appreciate your quick responses and the update. 
I checked all the options in the webcam menu, but the problem still stays on;
There may be something wrong with my system anyway.

![image](https://github.com/torinmb/mediapipe-touchdesigner/assets/135154493/0fdfd1a3-1f4a-478d-bd19-48fbf3c601ac)

**Comment by domisjustanumber** [comment]
Thanks for the screenshot - can you show me what the error messages say?

**Comment by sagarhz** [comment]
![image](https://github.com/torinmb/mediapipe-touchdesigner/assets/135154493/ebbe3e3d-9633-4f56-8c6c-21161dfd7a00)

**Comment by domisjustanumber** [comment]
Ah ok, it looks like you have something else running on network port 3002 that is blocking the plugom from working. Can you try another port number (3003 should do it) to see if that fixes it?

**Comment by sagarhz** [comment]
As I am a newbie, can u please guide me on how to do this?🫠

**Comment by torinmb** [comment]
<img width="1176" alt="Screenshot 2023-11-02 at 11 35 45 AM" src="https://github.com/torinmb/mediapipe-touchdesigner/assets/6014011/a7c3abaa-8a99-4a19-a167-281725638b58">
On the MediaPipe component try setting the MediaPipe Port to a different number. You may need to also hit the Reset button after changing it.

**Comment by sagarhz** [comment]
Thanks...
I'm shy to say it is still not working..

**Comment by torinmb** [comment]
<img width="1251" alt="Screenshot 2023-11-02 at 12 26 18 PM" src="https://github.com/torinmb/mediapipe-touchdesigner/assets/6014011/d830c0fd-8189-403a-84a9-5cd967c7e553">

Can you turn off the video device in and then try hitting reset on the MediaPipe component? Also if there are still errors can you send a screenshot of the error messages?

**Comment by torinmb** [comment]
Hey @sagarhz can you double check that you downloaded the release.zip? I get a similar error if I open the source.zip which is the development environment. Also make sure to extract the zip contents before opening. 
<img width="410" alt="Screenshot 2023-11-02 at 1 19 13 PM" src="https://github.com/torinmb/mediapipe-touchdesigner/assets/6014011/f88e6523-0758-4c9d-a766-8f3fa1036e67">

**Comment by torinmb** [comment]
Ok we found it!! We'll release an fix soon, but for now you can fix your network by doing the following:
1) Navigate to the MediaPipe component the Common tab 
2) Change the external tox path to ./toxes/MediaPipe.tox 
3) Click Re-Init Network
<img width="955" alt="Screenshot 2023-11-02 at 2 11 23 PM" src="https://github.com/torinmb/mediapipe-touchdesigner/assets/6014011/cd7bebc4-dc2c-4e4d-b820-7e94bb136571">

**Comment by torinmb** [comment]
I just pushed a quick fix in and added a new release.zip that fixes the issue. You can download the latest here: https://github.com/torinmb/mediapipe-touchdesigner/releases/tag/v0.3.2

It's still the same version number, but notice the release.zip was just updated

**Comment by sagarhz** [comment]
Hey! 
First of all, Sorry about my tardy response, and thank you for helping and trying to fix errors.
Second, YES!! It's working now 🥹🥹
Thanks a lot!!
I first downloaded the new release file, but the mediaPipe was still in the loading state. Then I tried the steps you mentioned about turning off my video device in, and it worked!
I don't know if the problem could be solved in the first place with just the video device in (as I was not online to check your response), but thank you for releasing a new version. I really appreciate your guidance.
![image](https://github.com/torinmb/mediapipe-touchdesigner/assets/135154493/496ca052-9adf-414b-abe9-ffbeac8cae55)
